modules/ocf/manifests/firewall/post.pp

```diff
@@ -28,10 +28,9 @@
 
 
   # Special devices we want to protect from most hosts
-  $devices = ['corruption-mgmt','hal-mgmt', 'jaws-mgmt', 'pagefault', 'pandemic-mgmt',
-              'papercut', 'radiation', 'riptide-mgmt']
-
-  $devices_ipv6 = ['radiation']
+  $devices_ipv4_only = ['corruption-mgmt','hal-mgmt', 'jaws-mgmt', 'pagefault',
+                        'pandemic-mgmt', 'papercut', 'riptide-mgmt']
+  $devices = ['radiation']
```

Comment:
this is data, not code -- would this fit better in hiera?

Reply:
I'd agree, hiera does seem like a better fit for this, I guess it would go in `common.yaml` currently, but we might need to split that up into different files if it gets too large or confusing. Currently it seems fine since it is short, but splitting it up later seems good, since it has a mixture of Mesos stuff and other common host settings that don't exactly belong together.

Reply:
Trouble is that rules in `ocf_kvm`, `ocf_desktop` and other places overriding these rules have hostnames hardcoded into the puppet file. Moving this data to hiera would break parallelism.

If we could have hiera rules applying to specific classes though, that would make a move to hiera much more desirable.